src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -2003,7 +2006,7 @@ bool
 shim::
 isAieRegistered()
 {
-  return (aieArray != nullptr);
+  return (m_aie_array != nullptr);
```

Comment:
Here the comparison can be argued to clarify the meaning of what it means to be registered, so this one I would leave.